tests/tt_metal/tt_metal/unit_tests_common/dram/test_dram_to_l1_multicast.cpp

```diff
@@ -130,3 +150,86 @@ TEST_F(CommonFixture, DRAMtoL1MulticastLoopbackSrc){
         ASSERT_TRUE(unit_tests_common::dram::test_dram_to_l1_multicast::dram_to_l1_multicast(this, devices_.at(id), test_config));
     }
 }
+TEST_F(CommonFixture, DRAMtoL1MulticastExcludeRegionUpLeft){
+    if (!getenv("TT_METAL_SLOW_DISPATCH_MODE")){
```

Comment:
why doesn't this work for FD?

Reply:
all other tests in this file are limited to slow dispatch only